homeassistant/components/flux/switch.py

```diff
@@ -329,6 +342,14 @@ def is_on(self):
                 time_state,
                 now,
             )
+            r_val, g_val, b_val = rgb
+            self._attributes = {}
+            self._attributes["Colour Temperature (K)"] = int(temp)
+            self._attributes["Brightness (%)"] = round(brightness / 255 * 100, 1)
+            self._attributes["rgb"] = (
+                str(int(r_val)) + ", " + str(int(g_val)) + ", " + str(int(b_val))
```

Comment:
Use f-strings instead of concatenating a string, see the [style guidelines](https://developers.home-assistant.io/docs/en/development_guidelines.html#use-new-style-string-formatting) for more information.

Reply:
Changed string concats to f-strings in bc1419bf4ac7e6b700b11f9fd59850419b10de56